resources/conf/log_config.toml

```diff
@@ -29,3 +29,6 @@ logLevel = "INFO"
 enable = false
 logfile = "/tmp/envoy.access.log" # This file will be created inside router container.
 format = "[%START_TIME%] '%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%' %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% '%REQ(X-FORWARDED-FOR)%' '%REQ(USER-AGENT)%' '%REQ(X-REQUEST-ID)%' '%REQ(:AUTHORITY)%' '%UPSTREAM_HOST%'\n"
+
+[debugLogs]
```

Comment:
IMHO, the logs you have added are debug logs and then enabling debug logs in envoy as usual would work. We won't need a new config then.

Reply:
we buffer the request/response body only if it is enabled from configs, so we need this config right? otherwise there might be a perf issue

Reply:
perf decrease is to be expected when debug logs are enabled, right?

Reply:
in envoy Lua code they do not support a way to check whether debug is enabled or not